docs/spark/tutorials/databricks-deployment.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Deploy a .NET for Apache Spark application to Databricks
+description: Discover how to deploy a .NET for Apache Spark application to Databricks.
+ms.date: 05/14/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to deployment .NET for Apache Spark application to Databricks.
+---
+
+# Deploy a .NET for Apache Spark application to Databricks
+
+This tutorial teaches how to deploy a .NET for Apache Spark application to Databricks.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+> * Prepare Microsoft.Spark.Worker
+> * Publish your Spark .NET app
+> * Deploy your app to Databricks
+> * Run your app
+
+## Prerequisites
+
+Before you start, make sure you have the following:
+
+* Download the [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html).
+* Download [install-worker.sh](https://github.com/dotnet/spark/blob/master/deployment/install-worker.sh) to your local machine. This is a helper script that you use later to copy .NET for Apache Spark dependent files into your Spark cluster's worker nodes.
+
+## Prepare worker dependencies
+
+**Microsoft.Spark.Worker** is a backend component that lives on the individual worker nodes of your Spark cluster. When you want to execute a C# UDF (user-defined function), Spark needs to understand how to launch the .NET CLR to execute the UDF. **Microsoft.Spark.Worker** provides a collection of classes to Spark that enable this functionality.
+
+1. Select a [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases) Linux netcoreapp release to be deployed on your cluster.
+
+   For example, if you want `.NET for Apache Spark v0.1.0` using `netcoreapp2.1`, you'd download [Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz](https://github.com/dotnet/spark/releases/download/v0.1.0/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz).
+
+2. Upload `Microsoft.Spark.Worker.<release>.tar.gz` and [install-worker.sh](https://github.com/dotnet/spark/blob/master/deployment/install-worker.sh) to a distributed file system (e.g., DBFS) that your cluster has access to.
+
+## Prepare your .NET for Apache Spark app
+
+1. Follow the [Get Started](get-started.md) tutorial to build your app.
+
+2. Publish your Spark .NET app as self-contained.
+
+   You can run the following command on Linux.
+
+   ```bash
+   foo@bar:~/path/to/app$ dotnet publish -c Release -f netcoreapp2.1 -r ubuntu.16.04-x64
```

Comment:
Same as the other article, and throughout... Does having a prompt make it harder to copy the command line?

Reply:
oh, I see what you  mean. I can remove it.